app/controllers/clearance/passwords_controller.rb

```diff
@@ -37,8 +40,14 @@ def update
 
   private
 
-  def deliver_email(user)
-    mail = ::ClearanceMailer.change_password(user)
+  def create_password_reset_for(user)
```

Comment:
SInce we're going to target this for 2.0, I think we should not worry about attr_accessible and use strong params here. What do you think?